build-system/app-index/test/test-file-list.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {expect} = require('chai');
+const {FileList} = require('../file-list');
+const {getBoundAttr, parseHtmlChunk} = require('./helpers');
+
+
+describe('devdash', () => {
+
+  describe('FileList', () => {
+
+    it('wraps', () => {
+      const root = parseHtmlChunk(FileList({
+        basepath: 'basepath',
+        fileSet: [],
+        selectModePrefix: '/',
+      }));
+
+      expect(root.className).to.equal('file-list-container');
+
+      const {firstElementChild} = root;
+
+      expect(firstElementChild.className).to.equal('wrap');
+    });
+
+    it('creates amp-list', () => {
+      const root = parseHtmlChunk(FileList({
+        basepath: 'basepath',
+        fileSet: [],
+        selectModePrefix: '/',
+      }));
+
+      const {length} = root.getElementsByTagName('amp-list');
```

Comment:
you can use `.to.have.length` on things that are array-like, so you could do

`expect(root.getElementsByTagName('amp-list')).to.have.length(1);`

Reply:
Missed those, thanks!